build.gradle

```diff
@@ -17,6 +17,7 @@ plugins {
 java {
     toolchain {
         languageVersion = JavaLanguageVersion.of(8)
+        vendor = JvmVendorSpec.AMAZON
```

Comment:
On latest master we don't specify a vendor

I had to do this here, because the Adoptium JDK has disappeared with 404